loadtest/loadtest.go

```diff
@@ -72,7 +73,13 @@ func (lt *LoadTester) addUser() error {
 	}
 	lt.status.NumUsers++
 	lt.status.NumUsersAdded++
-	controller, err := lt.newController(activeUsers+1, lt.statusChan)
+	userID := activeUsers + 1
+	// If specified by the config, we login with the same user again,
+	// to simulate multiple sessions.
```

Comment:
The comment needs updating. We now pick a user randomly from the ones that are already running.